packages/backend/src/runtime/node/ephemeral.js

```diff
@@ -0,0 +1,154 @@
+const fs = require('node:fs');
+const path = require('node:path');
+
+const { isPublishableKey } = require('@clerk/shared');
+
+async function createEphemeralAccount() {
+  const body = await postJSON('https://api.clerkstage.dev/v1/public/demo_instance');
```

Comment:
Once implemented: also fetch a claim token of some sort so we can build a URL to sign up and claim the instance.